core/app/models/spree/stock/simple_coordinator.rb

```diff
@@ -96,6 +102,16 @@ def allocate_inventory(availability_by_location)
         end
       end
 
+      def sort_availability(availability)
```

Comment:
Maybe add some docs on how sorting works.

Reply:
@BenMorganIO you mean on this specific method or in the guides?

Reply:
I think that if we document this change well in the PR and in the guides it would be enough

Reply:
Yeah, method docs plus guide docs would be great. I recently needed something like this.